frontend/middleware_panic.go

```diff
@@ -15,9 +14,13 @@ import (
 func MiddlewarePanic(w http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
 	defer func() {
 		if e := recover(); e != nil {
-			if logger, ok := r.Context().Value(ContextKeyLogger).(*slog.Logger); ok {
-				logger.Error(fmt.Sprintf("panic: %#v\n%s\n", e, string(debug.Stack())))
+			logger, ok := LoggerFromContext(r.Context())
+			if !ok {
+				logger = DefaultLogger()
```

Comment:
@mbarnes I think this is a bit better than before because we can now show the stack trace no matter if there is a logger in the context (something that should happen most of the time, but in the worst case we were not showing the trace if no longer was present) 🙂